cmake/modules/FindSphinx.cmake

```diff
@@ -0,0 +1,20 @@
+include(FindPackageHandleStandardArgs)
+
+find_package(Python3)
+if(PYTHON3_FOUND)
```

Comment:
In case it allows additional simplifications of this logic, I should note that the MaterialX project requires Python 3.6 or greater!

https://github.com/AcademySoftwareFoundation/MaterialX#supported-platforms